src/cms/utils/slug_utils.py

```diff
@@ -26,76 +61,77 @@ def generate_unique_slug(form_object, foreign_model):
     * :func:`cms.forms.events.event_translation_form.EventTranslationForm.clean_slug`
     * :func:`cms.forms.pois.poi_translation_form.POITranslationForm.clean_slug`
 
-    :param form_object: The form which contains the slug field
-    :type form_object: ~django.forms.Form
+    :param args: The supplied arguments list
+    :type args: list
 
-    :param foreign_model: If the form instance has a foreign key to another model (e.g. because it is a translation of
-                          a content-object), this paramaeter contains the model of the foreign related object.
-    :type foreign_model: ~django.db.models.Model
+    :param kwargs: The supplied keyword arguments
+    :type kwargs: dict
 
     :raises ~django.core.exceptions.ValidationError: When no slug is given and there is also no field which can be used
                                                      as fallback (either ``title`` or ``name``).
 
     :return: An unique slug identifier
     :rtype: str
     """
+    slug = kwargs["slug"]
+    foreign_model = kwargs["foreign_model"]
+    object_instance = kwargs["object_instance"]
+    fallback = kwargs["fallback"]
```

Comment:
```suggestion
    slug = kwargs.get("slug")
    foreign_model = kwargs.get("foreign_model")
    object_instance = kwargs.get("object_instance")
    fallback = kwargs.get("fallback")
```
While you're at it, I'd suggest the `get()`-syntax here as well, because then all values which are not given are just ``None`` instead of raising an exception.